src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -1171,17 +1171,19 @@ private void CleanUp()
 
             try
             {
-                // on Unix, we need to kill the process to ensure it terminates as Dispose() merely
+                // on Unix, we need to kill the process (if not running in background) to ensure it terminates as Dispose() merely
                 // closes the redirected streams and the processs does not exit on macOS.  However,
```

Comment:
1. The beginning of the comment mentions Unix but the end is about MacOS, can you please explain? Should it be something like "on some distributions, like MacOS"?
2. Typo and double space, I know this line was changed but it's part of the modified commit so why not fix it

```suggestion
                // closes the redirected streams and the process does not exit on macOS. However,
```